teps/0104-propagating-parameters.md

```diff
@@ -0,0 +1,638 @@
+---
+status: implementable
+title: Propagating Parameters
+creation-date: '2022-04-11'
+last-updated: '2022-04-11'
+authors:
+- '@jerop'
+- '@bobcatfish'
+replaces:
+- TEP-0023
```

Comment:
For reviewers, [TEP metadata](https://github.com/tektoncd/community/blob/main/teps/0001-tekton-enhancement-proposal-process.md#tep-metadata) as described in TEP-0001